management/publishSettings.go

```diff
@@ -7,7 +7,7 @@ import (
 	"fmt"
 	"io/ioutil"
 
-	"github.com/Azure/go-pkcs12"
+	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12"
```

Comment:
that's what happens when you do `godep save -r` which rewrites your paths IMHO. I don't think we need path rewriting here but let's leave it as is for consistency until somebody complains. :)